app/views/organizations/shared/_unique_species.html.erb

```diff
@@ -0,0 +1,3 @@
+<% @adoptable_unique_species.each do |species| %>
```

Comment:
And here, the rails way is to never use instance variables in partials. It should just be `adoptable_unique_species` and then you can pass that to the partial when you call the partial in 
app/views/organizations/home/index.html.erb
app/views/organizations/adopter_fosterer/dashboard/index.html.erb